### DISCUSSION TITLE
Huggingface Models

### DISCUSSION THREAD

**Discussion by jon-fox** [linto-ai/whisper-timestamped#201] [Ideas]
What would be the lift to make this usable with other STT models available on huggingface?

**Comment by Jeronymous** [comment]
This is out of scope for this package.
 `whisper-timestamped` is just a wrapper around `openai-whisper` (a package to run Whisper models, written by the model "makers", so a reference implementation).

But you can find a lot of things in `transformers` and `speechbrain` packages to run most of models on HuggingFace

**Reply by jon-fox** [reply]
yeah, i was aware of a lot of the stuff out there to run the models themselves, but I'm not finding a lot that gives the timestamped output so well like this project does
